engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -355,4 +361,66 @@ private static boolean isErrorLevelEnable()
     return false;
   }
 
+  /**
+   * Adds Logger Appender
+   * @param name Appender name
+   * @param className Appender class name
+   * @param args Appender args
+   */
+  public static void addAppender(String name, String className, Object args)
+  {
+    if (getAppendersNames().contains(name)) {
+      logger.warn("A logger appender with the name '{}' exists. Cannot add a new logger appender with the same name", name);
+    } else {
+      try {
+        Constructor<?> constructor = Class.forName(className).getConstructor(Object.class);
+        AppenderSkeleton appender = (AppenderSkeleton)constructor.newInstance(args);
+        appender.setName(name);
```

Comment:
Is the name used to only name the appender? It is not being used to find the appender in anyway? In that case can it be internally handled within LoggerUtil. In other words what is the needed for allowing specification of multiple appenders currently?